daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -45,28 +45,17 @@ class DecodeV1Spec
   val dummyModuleDName = DamlLf1.DottedName.newBuilder().addSegments(dummyModuleStr).build()
   val dummyModuleName = DottedName.assertFromString(dummyModuleStr)
 
-  private def dummyModule(version: LV.Minor, interningIdx: Int) = {
```

Comment:
@remyhaemmerle-da I forgot to fix the tests on the first round. They got actually simpler. Yay!

Reply:
:+1: